config/environments/test.rb

```diff
@@ -47,4 +47,10 @@
 
   #enables a few aliases - context, should, and should_eventually methods
   config.minitest_spec_rails.mini_shoulda = true
+
+  # High logger level results in less IOps & faster tests
+  config.log_level = :warn
+
+  # Use separate cache stores for parallel_tests
+  config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
```

Comment:
I was trying to work out if you have the plugin installed, shouldn't we let it use that rather than overriding the cache store here?